Library/src/Multiplayer/SpaceEntity.cpp

```diff
@@ -645,16 +671,41 @@ void SpaceEntity::Deserialise(IEntityDeserialiser& Deserialiser)
 
 void SpaceEntity::DeserialiseFromPatch(IEntityDeserialiser& Deserialiser)
 {
+	SpaceEntityUpdateFlags UpdateFlags = SpaceEntityUpdateFlags(0);
+
+	if (ShouldUpdateParent)
```

Comment:
It looks like we're testing for this value before we've actually deserialized the state of the value (in line 681)?

Reply:
Good spot. This is only working because it is using my local value

Reply:
Latest change looks good to me 🙌